.github/pull_request_template.md

```diff
@@ -1,27 +1,21 @@
 # Objective
 
-- Describe the objective or issue this PR addresses.
-- If you're fixing a specific issue, say "Fixes #X".
+Why are you making these changes?
+Link any associated issues with "Fixes #X" so they can be automatically closed.
+If this is a particularly complex PR, please add a clear list of the changes made.
 
-## Solution
+Discuss any background information that may be needed for other reviewers to understand the changes you've made.
+This may include related issues, previous discussion, or links to documentation.
 
-- Describe the solution used to achieve the objective above.
+## Feedback wanted
 
```

Comment:
👍 I like this section, maybe include an advice to use checkbox and not remove previous feedback requests ?
```suggestion
- [x] A requested feedback which has been addressed. Refrain from removing those items from the list to give a better understanding of the evolution of this Pull Request.
- [ ] A feedback not addressed yet.
```

Reply:
I really like this, but it adds length, so I'll lean against it.